src/lib/stores/sdk.ts

```diff
@@ -50,4 +35,25 @@ const sdkForProject = {
     users: new Users(clientProject)
 };
 
-export { sdkForConsole, sdkForProject, setProject };
+export const sdk = {
+    forConsole: {
+        client: clientConsole,
+        account: new Account(clientConsole),
+        avatars: new Avatars(clientConsole),
+        functions: new Functions(clientConsole),
+        health: new Health(clientConsole),
+        locale: new Locale(clientConsole),
+        projects: new Projects(clientConsole),
+        teams: new Teams(clientConsole),
+        users: new Users(clientConsole)
+    },
+    get forProject() {
+        const pathname = window.location.pathname;
+        const projectMatch = pathname.match(/project-([a-zA-Z0-9]+)/);
+        if (projectMatch && projectMatch[1] !== clientProject.config.project) {
+            clientProject.setProject(projectMatch[1]);
+        }
```

Comment:
Could we use the `page.params` store with `get` here to follow svelte's way?

Reply:
I tried it, but the page store starts as an empty object, even when the url is already available:

The following code:
![image](https://user-images.githubusercontent.com/26071571/224011821-84712545-2fbc-499a-b958-da0302db9c07.png)
Results in the following logs:
![image](https://user-images.githubusercontent.com/26071571/224011741-d609bf67-5e91-4960-8ee3-caaed9326d4a.png)

Meaning that the sdk would be unset in some initial requests.